bookserver/main.py

```diff
@@ -100,3 +102,18 @@ def auth_exception_handler(request: Request, exc: NotAuthenticatedException):
     Redirect the user to the login page if not logged in
     """
     return RedirectResponse(url="/auth/login")
+
+
+@app.exception_handler(ValidationError)
```

Comment:
This is really neat! I simplifies the validation code and is overall great.

Would you mind adding a links to the docs on this?